### DISCUSSION TITLE
5.3 Compatibility

### DISCUSSION THREAD

**Discussion by tdhavema** [midgen/uesvon#30] [General]
Hi me again, I don't see where my other discussion went. I see a few days ago you did exactly what you said and updated a few things, however I still receive this message if I try to build from IDE using the plugin in the engine. Am I being impatient and you need more time or am I doing something wrong?
![IDE](https://github.com/midgen/uesvon/assets/150241330/db92dc61-4df1-4f35-9f7f-db6f2a576c79)

**Comment by midgen** [comment]
Have you tried regenerating the project files, re-opening the visual studio solution, and launching the editor from there?

I submitted some fixes for packaging the plugin a couple of days ago.

There's also a packaged release of 5.3 for win64 now, although you shouldn't need it....

**Reply by tdhavema** [reply]
Yes as soon as I enable it in editor, regardless of updating any files or restarting VS, or clicking generate project files on the .uproject it always throws me this error 
![Editor1](https://github.com/midgen/uesvon/assets/150241330/6ac007b3-66f0-4070-9104-4ba30d6c066f)
![Editor2](https://github.com/midgen/uesvon/assets/150241330/1364cd90-4534-4aae-9bd8-a0dd2d378959)

I will go look for the packaged release thank you

**Comment by tdhavema** [comment]
The packaged release works just fine now, thank you so much this means so much to me. Eventually I will learn how to write this all myself, but I really do appreciate you sharing your knowledge with everyone.
![Works](https://github.com/midgen/uesvon/assets/150241330/f927d8e8-42c4-44c6-8bb8-1f1c3d274f6b)

**Comment by midgen** [comment]
No worries, glad people are finding it useful :)

I think may find some time to give it a proper overhaul and take to production standard soon.....enough people trying it out!

**Reply by tdhavema** [reply]
I've spent weeks researching this, and this is the best solution I've found, even tho I haven't used it yet. I read all your documentation and wiki, and I looked through some of the file code. Everyone else is kinda close to what you have here, but this is the most flexible from what I've read, unless I'm going to write my own Voxel stuff directly into Vulkan. Whatever you do, it is appreciated.
